ext/financialacls/CRM/Utils/Check/Component/FinancialTypeAcls.php

```diff
@@ -22,9 +22,8 @@ class CRM_Utils_Check_Component_FinancialTypeAcls extends CRM_Utils_Check_Compon
    */
   public static function checkFinancialAclReport() {
     $messages = [];
-    $ftAclSetting = Civi::settings()->get('acl_financial_type');
     $financialAclExtension = civicrm_api3('extension', 'get', ['key' => 'biz.jmaconsulting.financialaclreport', 'sequential' => 1]);
-    if ($ftAclSetting && (($financialAclExtension['count'] == 1 && $financialAclExtension['values'][0]['status'] != 'installed') || $financialAclExtension['count'] !== 1)) {
+    if (($financialAclExtension['count'] == 1 && $financialAclExtension['values'][0]['status'] !== 'installed') || $financialAclExtension['count'] !== 1) {
```

Comment:
@eileenmcnaughton Do we need to add a check hook in this extension https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_check/?

Reply:
so I had doubts about how valid this is now - it does depend on where we land with https://github.com/civicrm/civicrm-core/pull/28967 - ie do we 

a) do expensive, super thorough thing in in core & make the extension obsolete or
b) do the thing that performs well & is probably enough for most cases in core & recommend people use the extension for the expensive thing - I haven't  had any engagement from @JoeMurray or @seamuslee001 on that but I think the value of the check is grey enough that removing for now seems OK

Reply:
now that @seamuslee001 has merged https://github.com/civicrm/civicrm-core/pull/28967 this check can go